modules/social_features/social_core/social_core.api.php

```diff
@@ -159,15 +159,15 @@ function hook_social_content_type_alter(array &$page_to_exclude) {
 }
 
 /**
- * Provide method to allows extensions to use the new content style on a node.
+ * Provide a method to alter new content style on an entity.
  *
  * @param array $compatible_content_type_forms
- *   Array of the nodes.
+ *   Array of the form identifiers.
  *
- * @see social_core_form_node_form_alter()
+ * @see social_core_form_alter()
  * @ingroup social_core_api
  */
-function hook_social_core_compatible_content_forms(array &$compatible_content_type_forms) {
```

Comment:
@OH17L As I can see you are not changing anything in our hooks, but just fixing to the correct name because it was wrong here, right?

Reply:
Correct.